client/my-sites/upgrades/checkout-thank-you/index.jsx

```diff
@@ -99,6 +101,39 @@ var CheckoutThankYou = React.createClass( {
 		return null;
 	},
 
+	goBackUrl() {
+		var purchases;
+		if ( this.isDataLoaded() ) {
+			purchases = getPurchases( this.props );
+
+			function shouldGoBackToPlans() {
```

Comment:
Also, why define functions here instead of setting up flags like

```js
var shouldGoBackToPlans = purchases.some( isJetpackPremium ) ||
  purchases.some( isJetpackBusiness ) ||
  purchases.some( isPremium ) ||
  purchases.some( isBusiness );
```

If we're worried about not running several some() calls, this function should only be called once.